src/driver/DriverOptions.ts

```diff
@@ -79,4 +79,8 @@ export interface DriverOptions {
      */
     readonly tablesPrefix?: string;
 
+    /**
+     * Specify the domain for active directory connections
```

Comment:
Please add dot at the end of comment. Also please add note that this option is mssql-specific.